### ISSUE TITLE
image is not public

### ISSUE THREAD

**Issue by hacker-h** [fanyx/docker-tmserver#1]
```
 docker-compose up
Creating network "trackmania-server_default" with the default driver
Pulling xaseco (fanyx/xaseco:latest)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling xaseco (fanyx/xaseco:latest)...
ERROR: pull access denied for fanyx/xaseco, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
``

**Comment by SleepyPxnda** [comment]
Changing the image from ` fanyx/xaseco:latest` to `fanyx/docker-xaseco:latest` worked for me

**Comment by fanyx** [comment]
GitHub never notified me that people opened issues on my own repository... very interesting. Sorry for the lack of a response...

If it is of any concern to you, I reworked the entire project to include TMServer and Xaseco in one container from now on.
There will be new documentation in the coming days but I will close this issue as it relates to an old version of the project.

If you're still having issues with the new version feel free to open a new issue, I have enabled notifications on all activity on this repo now.
